src/AutoComplete/AutoComplete.js

```diff
@@ -123,6 +123,8 @@ class AutoComplete extends Component {
     /** @ignore */
     onBlur: PropTypes.func,
     /** @ignore */
+    onClose: PropTypes.func,
```

Comment:
That property should be documented keeping the ` /** @ignore */` comment is making it hidden in the documentation autogenerated.